client/ayon_maya/api/lib.py

```diff
@@ -3678,7 +3678,7 @@ def dgcontext(mtime):
                                                                 None)
                     if dependency_visible is None:
                         mplug = get_visibility_mplug(dependency)
-                        dependency_visible = mplug.asBool(context)
+                        dependency_visible = mplug.asBool()
```

Comment:
is `context` in the line 3674 being used anywhere. Otherwise, it could be just like this below:
```
        with dgcontext(mtime):
```